apps/upload-log/upload-log-cf.yml

```diff
@@ -7,11 +7,16 @@ Parameters:
 
 Outputs:
 
-  LambdaArn:
-    Value: !GetAtt Lambda.Arn
+  UploadLogActivity:
```

Comment:
I'd consider calling this `ActivityArn`. `UploadLog` is probably redundant from context, and it specifies the value is a full ARN (as opposed to a name).